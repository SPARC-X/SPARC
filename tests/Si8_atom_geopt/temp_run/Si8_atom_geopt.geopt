:RELAXSTEP: 1
:E(Ha): -3.141018202136997E+01
:R(Bohr):
   0.100000000000000    0.210000000000000    0.170000000000000
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
  -0.015833752396495   -0.019579339096844   -0.017583628716224
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000   -0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000   -0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -9.9350930314E+00  -1.7122532967E+00  -1.9766366224E+00 
 -1.7122532967E+00  -1.0068731969E+01  -1.3615918242E+00 
 -1.9766366224E+00  -1.3615918242E+00  -1.0018819632E+01
:RELAXSTEP: 2
:E(Ha): -3.141033429523456E+01
:R(Bohr):
   0.097422241457847    0.206812454347981    0.167137358979080
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
  -0.015401963151073   -0.019277540122345   -0.017247533881615
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000   -0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000   -0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -9.9450789055E+00  -1.6701337133E+00  -1.9330170861E+00 
 -1.6701337133E+00  -1.0072045842E+01  -1.3185734611E+00 
 -1.9330170861E+00  -1.3185734611E+00  -1.0024712016E+01
:RELAXSTEP: 3
:E(Ha): -3.141316442204161E+01
:R(Bohr):
   0.034032691891395    0.127472270689669    0.096152031124198
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
  -0.005618402056596   -0.011804729897053   -0.009318536220438
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000   -0.000000000000000
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000   -0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0138702015E+01  -7.8362912811E-01  -1.0030504296E+00 
 -7.8362912811E-01  -1.0159446987E+01  -4.2393086382E-01 
 -1.0030504296E+00  -4.2393086382E-01  -1.0151949618E+01
:RELAXSTEP: 4
:E(Ha): -3.141442284253619E+01
:R(Bohr):
  10.183613131012741    0.021037643460205    0.012211012852047
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
   0.001490583235754   -0.001939974774007   -0.001078496754233
   0.000000000000000    0.000000000000000    0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000   -0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000   -0.000000000000000    0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0232251646E+01  -8.2072520557E-02  -1.4625516661E-01 
 -8.2072520557E-02  -1.0232294260E+01   1.1270382644E-01 
 -1.4625516661E-01   1.1270382644E-01  -1.0232218685E+01
:RELAXSTEP: 5
:E(Ha): -3.141445509068068E+01
:R(Bohr):
  10.188304338185139    0.004295559983638    0.001282761249004
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
   0.001101959813912   -0.000401900394480   -0.000110487047433
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000    0.000000000000000   -0.000000000000000
   0.000000000000000   -0.000000000000000   -0.000000000000000
  -0.000000000000000   -0.000000000000000    0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0234823527E+01  -8.4300063251E-03  -3.0119793969E-02 
 -8.4300063251E-03  -1.0234793903E+01   8.2446781617E-02 
 -3.0119793969E-02   8.2446781617E-02  -1.0234789299E+01
:RELAXSTEP: 6
:E(Ha): -3.141446151880450E+01
:R(Bohr):
  10.195757864025197   10.199226169050014   10.198791167475630
   2.550000000000000    2.550000000000000    2.550000000000000
   0.000000000000000    5.100000000000000    5.100000000000000
   2.550000000000000    7.650000000000000    7.650000000000000
   5.100000000000000    0.000000000000000    5.100000000000000
   7.650000000000000    2.550000000000000    7.650000000000000
   5.100000000000000    5.100000000000000    0.000000000000000
   7.650000000000000    7.650000000000000    2.550000000000000
:F(Ha/Bohr):
   0.000400509135298    0.000072049085888    0.000113533818573
  -0.000000000000000   -0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000    0.000000000000000    0.000000000000000
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000    0.000000000000000   -0.000000000000000
   0.000000000000000    0.000000000000000    0.000000000000000
  -0.000000000000000   -0.000000000000000    0.000000000000000
:CELL:   1.0200000000E+01   1.0200000000E+01   1.0200000000E+01
:VOLUME:   1.0612080000E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.0235282260E+01   8.4834831913E-03   5.3908678577E-03 
  8.4834831913E-03  -1.0235278003E+01   2.9911508042E-02 
  5.3908678577E-03   2.9911508042E-02  -1.0235278131E+01
