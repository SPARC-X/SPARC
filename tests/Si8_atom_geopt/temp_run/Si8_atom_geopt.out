***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:41:19 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 102 102 102
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 50
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
RELAX_FLAG: 1
RELAX_METHOD: LBFGS
RELAX_NITER: 300
L_HISTORY: 20
L_FINIT_STP: 0.005
L_MAXMOV: 0.2
L_AUTOSCALE: 1
L_LINEOPT: 1
L_ICURV: 1
CALC_STRESS: 1
TWTIME: 1E+09
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 1.00E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 5
TOL_RELAX: 5.00E-04
PRINT_RELAXOUT: 1
OUTPUT_FILE: Si8_atom_geopt
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.1 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8_atom_geopt.out
Relax output printed to            :  Si8_atom_geopt.geopt
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
Pseudocharge radii of atom type 1  :  6.50 6.50 6.50 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  1.36 GB
Estimated memory per processor     :  58.02 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.8358632510E+00        2.030E-01        15.001
2            -3.9136964948E+00        1.956E-01        4.723
3            -3.9258982653E+00        1.803E-01        4.827
4            -3.9265713998E+00        1.379E-01        4.906
5            -3.9256574294E+00        5.381E-02        4.777
6            -3.9259036551E+00        3.500E-02        4.794
7            -3.9260260521E+00        1.582E-02        4.728
8            -3.9261566747E+00        1.304E-02        4.748
9            -3.9262247103E+00        5.308E-03        4.648
10           -3.9262477147E+00        2.915E-03        4.626
11           -3.9262596437E+00        1.731E-03        4.609
12           -3.9262655140E+00        9.804E-04        4.541
13           -3.9262687507E+00        6.900E-04        4.532
14           -3.9262705029E+00        3.310E-04        4.461
15           -3.9262715115E+00        2.248E-04        4.404
16           -3.9262720614E+00        1.925E-04        4.303
17           -3.9262723729E+00        3.151E-04        4.383
18           -3.9262725750E+00        3.453E-04        4.324
19           -3.9262726691E+00        3.638E-04        4.290
20           -3.9262727176E+00        3.965E-04        4.306
21           -3.9262727468E+00        3.931E-04        4.349
22           -3.9262727700E+00        3.560E-04        4.335
23           -3.9262727863E+00        3.330E-04        4.370
24           -3.9262727894E+00        4.624E-04        4.380
25           -3.9262727878E+00        3.771E-04        4.275
26           -3.9262727876E+00        3.958E-04        4.349
27           -3.9262728039E+00        3.432E-04        4.303
28           -3.9262727907E+00        3.859E-04        4.405
29           -3.9262727801E+00        1.332E-04        4.345
30           -3.9262727779E+00        7.435E-05        4.328
31           -3.9262727746E+00        3.703E-05        4.324
32           -3.9262727610E+00        1.742E-05        4.275
33           -3.9262727807E+00        1.047E-05        4.147
34           -3.9262727769E+00        7.051E-06        4.186
35           -3.9262727799E+00        2.240E-06        4.035
36           -3.9262727798E+00        1.705E-06        4.094
37           -3.9262727825E+00        7.015E-07        3.330
Total number of SCF: 37    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000351606222403.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9262727527E+00 (Ha/atom)
Total free energy                  : -3.1410182021E+01 (Ha)
Band structure energy              :  1.6587128474E+00 (Ha)
Exchange correlation energy        : -9.8386879363E+00 (Ha)
Self and correction energy         : -1.0219605145E+02 (Ha)
Entropy*kb*T                       : -3.8612814731E-03 (Ha)
Fermi level                        :  2.4284686924E-01 (Ha)
RMS force                          :  3.8390311140E-03 (Ha/Bohr)
Maximum force                      :  3.0712248912E-02 (Ha/Bohr)
Time for force calculation         :  0.179 (sec)
Pressure                           :  1.0007548211E+01 (GPa)
Maximum stress                     :  1.0068731969E+01 (GPa)
Time for stress calculation        :  0.437 (sec)
Relax step time                    :  176.227 (sec)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9262938721E+00        2.602E-03        4.793
2            -3.9262924887E+00        1.400E-03        4.545
3            -3.9262917950E+00        4.398E-04        4.433
4            -3.9262917812E+00        2.128E-04        4.316
5            -3.9262917786E+00        1.046E-04        4.324
6            -3.9262917784E+00        2.628E-05        4.150
7            -3.9262917831E+00        1.621E-05        4.234
8            -3.9262917721E+00        8.393E-06        4.188
9            -3.9262917714E+00        3.909E-06        4.076
10           -3.9262917949E+00        1.794E-06        3.986
11           -3.9262917881E+00        1.202E-06        3.965
12           -3.9262917804E+00        6.336E-07        3.268
Total number of SCF: 12    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000372399731341.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9262917869E+00 (Ha/atom)
Total free energy                  : -3.1410334295E+01 (Ha)
Band structure energy              :  1.6598597118E+00 (Ha)
Exchange correlation energy        : -9.8391980125E+00 (Ha)
Self and correction energy         : -1.0219605141E+02 (Ha)
Entropy*kb*T                       : -3.7962037396E-03 (Ha)
Fermi level                        :  2.4289032326E-01 (Ha)
RMS force                          :  3.7631466369E-03 (Ha/Bohr)
Maximum force                      :  3.0105173095E-02 (Ha/Bohr)
Time for force calculation         :  0.177 (sec)
Pressure                           :  1.0013945588E+01 (GPa)
Maximum stress                     :  1.0072045842E+01 (GPa)
Time for stress calculation        :  0.435 (sec)
Relax step time                    :  52.254 (sec)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9277613630E+00        5.664E-02        4.941
2            -3.9270862035E+00        3.332E-02        4.790
3            -3.9266493648E+00        1.028E-02        4.651
4            -3.9266400288E+00        6.238E-03        4.594
5            -3.9266434264E+00        2.455E-03        4.574
6            -3.9266455298E+00        1.215E-03        4.514
7            -3.9266453723E+00        4.366E-04        4.388
8            -3.9266454880E+00        3.033E-04        4.377
9            -3.9266455421E+00        1.141E-04        4.267
10           -3.9266455452E+00        6.410E-05        4.294
11           -3.9266455403E+00        3.550E-05        4.230
12           -3.9266455476E+00        1.704E-05        4.166
13           -3.9266455498E+00        1.063E-05        4.180
14           -3.9266455468E+00        4.729E-06        4.094
15           -3.9266455535E+00        4.664E-06        4.051
16           -3.9266455565E+00        2.194E-06        4.064
17           -3.9266455463E+00        1.639E-06        3.971
18           -3.9266455491E+00        9.044E-07        3.270
Total number of SCF: 18    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.001489500019993.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9266455528E+00 (Ha/atom)
Total free energy                  : -3.1413164422E+01 (Ha)
Band structure energy              :  1.6819551129E+00 (Ha)
Exchange correlation energy        : -9.8492282200E+00 (Ha)
Self and correction energy         : -1.0219605058E+02 (Ha)
Entropy*kb*T                       : -2.3876976162E-03 (Ha)
Fermi level                        :  2.4367926624E-01 (Ha)
RMS force                          :  2.0068367791E-03 (Ha/Bohr)
Maximum force                      :  1.6054694233E-02 (Ha/Bohr)
Time for force calculation         :  0.176 (sec)
Pressure                           :  1.0150032873E+01 (GPa)
Maximum stress                     :  1.0159446987E+01 (GPa)
Time for stress calculation        :  0.434 (sec)
Relax step time                    :  79.621 (sec)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9273675518E+00        1.247E-01        4.830
2            -3.9270344268E+00        8.849E-02        4.722
3            -3.9266016549E+00        2.000E-02        4.504
4            -3.9266777876E+00        1.155E-02        4.559
5            -3.9267733963E+00        4.255E-03        4.513
6            -3.9267990911E+00        1.769E-03        4.446
7            -3.9268020444E+00        8.692E-04        4.343
8            -3.9268025536E+00        2.321E-04        4.242
9            -3.9268027998E+00        1.094E-04        4.250
10           -3.9268028577E+00        6.173E-05        4.158
11           -3.9268028435E+00        2.490E-05        4.137
12           -3.9268028332E+00        1.859E-05        4.047
13           -3.9268028503E+00        1.294E-05        3.947
14           -3.9268028466E+00        1.091E-05        3.941
15           -3.9268028499E+00        4.550E-06        3.994
16           -3.9268028443E+00        4.219E-06        3.912
17           -3.9268028510E+00        1.439E-06        3.862
18           -3.9268028440E+00        7.708E-07        3.263
Total number of SCF: 18    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.006354335006937.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9268028553E+00 (Ha/atom)
Total free energy                  : -3.1414422843E+01 (Ha)
Band structure energy              :  1.6928562446E+00 (Ha)
Exchange correlation energy        : -9.8544171629E+00 (Ha)
Self and correction energy         : -1.0219605014E+02 (Ha)
Entropy*kb*T                       : -1.5227839128E-03 (Ha)
Fermi level                        :  2.4386784272E-01 (Ha)
RMS force                          :  3.3420838735E-04 (Ha/Bohr)
Maximum force                      :  2.6736670988E-03 (Ha/Bohr)
Time for force calculation         :  0.176 (sec)
Pressure                           :  1.0232254864E+01 (GPa)
Maximum stress                     :  1.0232294260E+01 (GPa)
Time for stress calculation        :  0.432 (sec)
Relax step time                    :  77.944 (sec)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9268008855E+00        1.975E-03        4.580
2            -3.9268044561E+00        1.336E-03        4.393
3            -3.9268065133E+00        6.150E-04        4.411
4            -3.9268068559E+00        2.495E-04        4.231
5            -3.9268068406E+00        1.324E-04        4.231
6            -3.9268068591E+00        6.039E-05        4.197
7            -3.9268068529E+00        2.043E-05        4.127
8            -3.9268068812E+00        1.396E-05        4.066
9            -3.9268068784E+00        6.153E-06        3.981
10           -3.9268068850E+00        4.015E-06        3.908
11           -3.9268068807E+00        1.865E-06        3.858
12           -3.9268068863E+00        1.357E-06        3.877
13           -3.9268068919E+00        5.735E-07        3.265
Total number of SCF: 13    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.007418877721120.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9268068863E+00 (Ha/atom)
Total free energy                  : -3.1414455091E+01 (Ha)
Band structure energy              :  1.6931624164E+00 (Ha)
Exchange correlation energy        : -9.8545661388E+00 (Ha)
Self and correction energy         : -1.0219605013E+02 (Ha)
Entropy*kb*T                       : -1.4960642542E-03 (Ha)
Fermi level                        :  2.4386580741E-01 (Ha)
RMS force                          :  1.4726924462E-04 (Ha/Bohr)
Maximum force                      :  1.1781539569E-03 (Ha/Bohr)
Time for force calculation         :  0.173 (sec)
Pressure                           :  1.0234802243E+01 (GPa)
Maximum stress                     :  1.0234823527E+01 (GPa)
Time for stress calculation        :  0.431 (sec)
Relax step time                    :  55.123 (sec)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9268060493E+00        3.410E-04        4.369
2            -3.9268069871E+00        2.190E-04        4.098
3            -3.9268073850E+00        1.439E-04        4.139
4            -3.9268076368E+00        6.368E-05        4.204
5            -3.9268076811E+00        2.807E-05        4.071
6            -3.9268076861E+00        1.473E-05        4.055
7            -3.9268077028E+00        8.065E-06        3.953
8            -3.9268076998E+00        2.892E-06        3.895
9            -3.9268076902E+00        2.530E-06        3.806
10           -3.9268076782E+00        1.959E-06        3.895
11           -3.9268076959E+00        8.873E-07        3.267
Total number of SCF: 11    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.008059669331808.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9268076899E+00 (Ha/atom)
Total free energy                  : -3.1414461519E+01 (Ha)
Band structure energy              :  1.6932208784E+00 (Ha)
Exchange correlation energy        : -9.8545945513E+00 (Ha)
Self and correction energy         : -1.0219605013E+02 (Ha)
Entropy*kb*T                       : -1.4909822576E-03 (Ha)
Fermi level                        :  2.4386553029E-01 (Ha)
RMS force                          :  5.2809883988E-05 (Ha/Bohr)
Maximum force                      :  4.2247907190E-04 (Ha/Bohr)
Time for force calculation         :  0.173 (sec)
Pressure                           :  1.0235279465E+01 (GPa)
Maximum stress                     :  1.0235282260E+01 (GPa)
Time for stress calculation        :  0.436 (sec)
Relax step time                    :  45.692 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  486.935 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
